.travis.yml

```diff
@@ -1,6 +1,17 @@
 sudo: false
 language: groovy
```

Comment:
For those following along at home, that doesn't work. It just picks the first item.